src/mad_spark_alt/core/simple_qadi_orchestrator.py

```
@@ -448,8 +463,8 @@ def _parse_hypothesis_scores(
             if in_section:
                 if (
                     re.match(rf"^(?:-\s*)?H{hypothesis_num + 1}:", line)
-                    or line.startswith("ANSWER:")
-                    or line.startswith("Action Plan:")
+                    or line.startswith(ANSWER_PREFIX)
+                    or line.startswith(ACTION_PLAN_PREFIX)
```

Ruff (0.12.2)

Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)